### ISSUE TITLE
Add another **OR** listing strategy to add +5% +10% +15% from users buying price

### ISSUE THREAD

**Issue by yangwao** [kodadot/nft-gallery#7105]
Probably for follow-up, we can add another **OR** strategy to add +5% +10% +15% from users buying price.

![image](https://github.com/kodadot/nft-gallery/assets/5887929/d765d331-b56a-4435-ba34-ae1de5f62380)

in this case if buy was 0.6KSM, 15% add up should be 0.69KSM
![image](https://github.com/kodadot/nft-gallery/assets/5887929/6210573f-72d5-41cf-b508-d5e9e235c73b)

_Originally posted by @yangwao in https://github.com/kodadot/nft-gallery/issues/7076#issuecomment-1703805462_

**Comment by exezbcz** [comment]
yes sir on it

**Comment by stephenjason89** [comment]
@yangwao @exezbcz  would you want this to be accumulated instead. Like first click of +5% will add 5 then next click will add 5more?

**Comment by yangwao** [comment]
> Like first click of +5% will add 5 then next click will add 5more?

good idea tbh, we can keep it for another iteration. On mobile make lot of sense to do tap tap for setting price

**Comment by stephenjason89** [comment]
@yangwao implemented it already ♥️

**Comment by exezbcz** [comment]
@stephenjason89 I think there should be some design changes if we want it to work like that. I will also create a follow-up for the buying price and this so it is clearer what it does. Thanks for implementing it!

**Comment by stephenjason89** [comment]
@exezbcz Please check the latest if that would be ok for you. Thank you so much
